user/incus/template.py

```diff
@@ -0,0 +1,81 @@
+pkgname = "incus"
+pkgver = "6.4.0"
+pkgrel = 0
+build_style = "go"
+make_build_args = ["./..."]
+hostmakedepends = [
+    "automake",
+    "bash",
+    "git",
+    "go",
+    "go-md2man",
+    "libtool",
+    "pkgconf",
+]
+makedepends = [
+    "acl-devel",
+    "cowsql-devel",
+    "libatomic-chimera-devel-static",
+    "libseccomp-devel",
+    "libcap-devel",
+    "linux-headers",
+    "libuv-devel",
+    "libunwind-devel-static",
+    "lxc-devel",
+    "musl-devel-static",
+    "raft-devel",
+    "sqlite-devel",
+    "udev-devel",
+]
+depends = [
+    "acl",
+    "acl-progs",
+    "attr-progs",
+    "dnsmasq",
+    "iptables",
+    "libvirt",
+    "lxc",
+    "rsync",
+    "squashfs-tools",
+    "util-linux",
+    "xz",
+]
+go_build_tags = ["libsqlite3"]
+pkgdesc = "Powerful system container and virtual machine manager"
+maintainer = "tj <tjheeta@gmail.com>"
+license = "Apache-2.0"
+url = "https://github.com/lxc/incus"
+source = f"{url}/archive/v{pkgver}.tar.gz"
+sha256 = "c6ad8260a4954b0e4f77de66220d5450290b57858f8c16ee83f12996eea86db4"
+# void ignores TestConvertNetworkConfig
+# https://github.com/void-linux/void-packages/blob/7dd18621834b9bdb2d4e5b081c820d24fa3180dd/srcpkgs/incus/template#L40
+# go test -v -tags libsqlite3 -skip TestConvertNetworkConfig ./...
+options = ["!check"]
+
+
+def post_build(self):
+    # Build the agent statically
+    self.env["CGO_ENABLED"] = "1"
+    self.env["CGO_LDFLAGS"] = "-static-pie"
+    self.go_build_tags = ["agent", "netgo", "osusergo"]
+    self.make_build_args = ["./..."]
+    self.golang.build(wrksrc="cmd/incus-agent")
```

Comment:
i don't think you're supposed to modify template attributes inside template functions except for `init_*`, but i'm not quite sure how to do this better

Reply:
Is there anything for me to do on this? Shall I resolve?